docs/content/drive.md

```diff
@@ -1814,10 +1815,7 @@ then select "OAuth client ID".
    If you chose "Internal" you don't need to publish and can skip straight to
    Step 11 but your destination drive must be part of the same Google Workspace.)
 
-10. Go to "Oauth consent screen" and then click "PUBLISH APP" button and confirm.
-   You will also want to add yourself as a test user.
-
-11. Provide the noted client ID and client secret to rclone.
+10. Provide the noted client ID and client secret to rclone.
```

Comment:
I'd prefer it that you left this step but put a note that it isn't necessary for such and such a reason. How does that sound?

Reply:
Sounds OK for me. My thoughts would be that Google might reject and disable apps for some people if purely this publish becomes a normal step in any situation, not only rclone, as per too many requests that never end in paid projects for them.

Reply:
Actually, we can forget about this. I did not know about expiring grant.